.github/workflows/ci-synthetic-test-trigger.yml

```
@@ -0,0 +1,71 @@
+            const prNumber = context.payload.pull_request?.number || parseInt(core.getInput('pr_number'));
+            const { data: pr } = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: prNumber
+            });
+
+            const syntheticTestMatch = pr.body?.match(/\[(?<checked>[ x])\]\s+\*\*Synthetic Test Environment\*\* <!-- #synthetic-test -->/);
+            const shouldRun = syntheticTestMatch?.groups.checked === 'x';
+
+            const targetUrlMatch = pr.body?.match(/\*\*Target URL:\*\*\s*(https?:\/\/[^\s\n]+)/i);
+            const targetUrl = targetUrlMatch?.[1] 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)